docs/interfaces.md

```diff
@@ -249,6 +249,13 @@ Can configure network firewalling giving privileged access to networking.
 
 * Auto-Connect: no
 
+### fuse
+
+Can mount fuse filesystems (as root only).
+
+Usage: common
+Auto-Connect: yes
+
```

Comment:
Can you update this for the format in the recently committed interface documentation improvements PR?